packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -520,11 +520,8 @@ mixin SchedulerBinding on BindingBase {
   /// Returns true if a task was executed and there are other tasks remaining
```

Comment:
Update this as well to indicate that it also returns true if no task executed, but other tasks are remaining?